pkg/apis/certmanager/v1alpha2/types_issuer.go

```diff
@@ -293,6 +293,11 @@ type ACMEChallengeSolverHTTP01Ingress struct {
 	// used for HTTP01 challenges
 	// +optional
 	PodTemplate *ACMEChallengeSolverHTTP01IngressPodTemplate `json:"podTemplate,omitempty"`
+
+	// Optional additional annotations to add to Ingress resources used to solve ACME
+	// challenges that use this challenge solver.
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
Instead of adding this field here directly, can you create an `IngressTemplate` field in a similar manner to the `PodTemplate` above?